dcs-packaging-tool/dcs-packaging-tool-impl/src/test/java/org/dataconservancy/packaging/tool/impl/GeneralPackageDescriptionCreatorTest.java

```diff
@@ -32,15 +32,11 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Oh sure my bad, I forgot to change the IntelliJ setting not to do it. I would have to do it later today or you could merge and just make that one change. Your decision.